values.yaml

```diff
@@ -82,6 +82,7 @@ global:
   #nodeExporterEnabled: true
 
   prometheusPostgresExporterEnabled: false
+  prometheusBlackboxExporterEnabled: false
```

Comment:
There is an already existing flag
https://github.com/astronomer/astronomer/blob/ea401718d2393651e7cddff3011daf1565891cca/values.yaml#L87

Reply:
using this flag.